shell/browser/api/electron_api_base_window.cc

```diff
@@ -123,11 +123,6 @@ BaseWindow::BaseWindow(gin_helper::Arguments* args,
 BaseWindow::~BaseWindow() {
   CloseImmediately();
 
-  // Destroy the native window in next tick because the native code might be
-  // iterating all windows.
-  base::SingleThreadTaskRunner::GetCurrentDefault()->DeleteSoon(
```

Comment:
This regresses https://github.com/electron/electron/issues/42975

Reply:
Fixed in: https://github.com/electron/electron/pull/44628/commits/7c36ddeddbcd79e6d930f018b8dd08f92cbc32a2

Reply:
Is fix from https://github.com/electron/electron/commit/7c36ddeddbcd79e6d930f018b8dd08f92cbc32a2 good enough?

Reply:
ping